### ISSUE TITLE
Parsing error in Admin settings tab

### ISSUE THREAD

**Issue by SofiaSazonova** [data-dot-all/dataall#1481]
### Describe the bug

```
Cannot read properties of null (reading 'name')
TypeError: Cannot read properties of null (reading 'name')
```

### How to Reproduce

Admin Settings `/console/administration`, tab `TEAMS`, on click `Permissions`

### OS

macOs Sonoma 14.2

### Python version

3.11

### AWS data.all version

2.6.0

**Comment by SofiaSazonova** [comment]
The problem is here 
```
{
    "data": {
        "listTenantPermissions": [
            {
                "name": "MANAGE_GROUPS",
                "description": "Manage teams",
                "__typename": "Permission"
            },
            {
                "name": "MANAGE_ENVIRONMENTS",
                "description": "Manage environments",
                "__typename": "Permission"
            },
            {
                "name": "MANAGE_ORGANIZATIONS",
                "description": "Manage organizations",
                "__typename": "Permission"
            },
            {
                "name": "MANAGE_SGMSTUDIO_USERS",
                "description": "Manage SageMaker Studio users",
                "__typename": "Permission"
            },
            {
                "name": "MANAGE_NOTEBOOKS",
                "description": "Manage notebooks",
                "__typename": "Permission"
            },
            {
                "name": "MANAGE_GLOSSARIES",
                "description": "Manage glossaries",
                "__typename": "Permission"
            },
            {
                "name": "MANAGE_PIPELINES",
                "description": "Manage pipelines",
                "__typename": "Permission"
            },
            {
                "name": "MANAGE_DATASETS",
                "description": "Manage datasets",
                "__typename": "Permission"
            },
            null, <-----HERE
            null, <-----HERE
            {
                "name": "MANAGE_DASHBOARDS",
                "description": "Manage dashboards",
                "__typename": "Permission"
            },
            {
                "name": "MANAGE_WORKSHEETS",
                "description": "Manage worksheets",
                "__typename": "Permission"
            }
        ]
    }
}
```

**Comment by SofiaSazonova** [comment]
Problem was in my development account. 
My DB missed MANAGE_REDSHIFT_* permissions. The reason behind it is:
1. I tested my PRs in my account, particularly migrations, so `alembic_version` in my acc was `X`
2. Before my PR was merged I merged DB migrations, so my `revision X` became the last one, after redshift migration.
3. Since `X` was already in the `alembic_version`, upon update of my dev acc, previous migrations were not processed.

Be smart, mind your migrations ;)
Anyway, this bugfix must be useful. At least we should not have any frontend problems.
